src/browser/urls.py

```diff
@@ -65,9 +65,9 @@
         name='browser-table_delete'),
     url(r'^delete/(\w+)/(\w+)/card/(\w+)/?$', 'browser.views.card_delete',
         name='browser-card_delete'),
-
     url(r'^delete/(\w+)/(\w+)/file/([ -~]+)/?$', 'browser.views.file_delete',
         name='browser-file_delete'),
+    url(r'^delete/collaborator/(\w+)/(\w+)/(\w+)/?$', 'browser.views.repo_delete_collaborator'),
```

Comment:
This should be assigned a name. Take a look at the naming conventions in the same file.